core/src/main/java/hudson/Util.java

```diff
@@ -1624,6 +1626,21 @@ public static Properties loadProperties(@Nonnull String properties) throws IOExc
         return p;
     }
 
+    /**
+     * @return The list of strings of computer names (excluding master)
+     * @since TODO
+     */
+    @Nonnull
+    public static List<String> getComputerNames() {
```

Comment:
I do not think the method belongs to `Util`, perhaps `Nodes` is a better option. Also, the clients seem to care more about nodes than computers.